### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by lucaswloch**

### CODE REVIEW

File: staff/lucas-wloch/challenges/calculadora-app/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Calculator App
+## INTRO
+
+
+This is a basic web calculator app with all the normal functions.
+You can do sumas, restas, multiplicaciones y divisiones.
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/calculadora-app/README.md]: everything is ok, but you should use just one language. Thanks

File: staff/lucas-wloch/challenges/calculadora-app/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EJ Calculadora</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="calculate.js" defer></script>
+    <script src="main.js" defer></script>
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/calculadora-app/index.html]: script src=... should be at the end, just before body closes

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        this.clear()
+    }
+
+    clear(){
+        this.textActual = ""
+        this.textAnterior = ""
+        this.operation = undefined
+        textAnterior.innerText = ""
+        textActual.innerText = ""
+
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/calculadora-app/calculate.js]: you can remove this empty line

File: staff/lucas-wloch/challenges/calculadora-app/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+### documentacion del proyecto
```
Comment (manuelbarzi) [inline staff/lucas-wloch/challenges/calculadora-app/README.md]: in english please

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
+class calculator {
+    constructor(textAnterior,textActual) {
+        this.textAnterior = textAnterior
```
Comment (manuelbarzi) [inline staff/lucas-wloch/challenges/calculadora-app/calculate.js]: you need to set all semicolons after your statements

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        const current = parseFloat(this.textActual)
+        if (isNaN(prev) || isNaN(current)) return
+        switch (this.operation) {
+
+            case "+":
+                computation = prev + current
+                break
+            case "-":
+                computation = prev - current
+                break
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/calculadora-app/calculate.js]: faltan puntos y comas

File: staff/lucas-wloch/challenges/memo-test-app/main.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (manuelbarzi) [inline staff/lucas-wloch/challenges/memo-test-app/main.js]: rename folder to memory-game-app

File: staff/lucas-wloch/challenges/calculadora-app/main.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+
+
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/calculadora-app/main.js:9]: ; are missing

File: staff/lucas-wloch/challenges/calculadora-app/main.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+const numberButtons = document.querySelectorAll('[data-number]')
+const operationButtons = document.querySelectorAll('[data-operator]')
+const equalsButton = document.querySelector('[data-equals]')
+const clearButton = document.querySelector('[data-limpiar]')
+const textAnterior = document.querySelector('[data-text-anterior]')
+const textActual = document.querySelector('[data-text-actual]')
+const backButton = document.querySelector("[data-back]")
+
+
+ calculator = new calculator(textAnterior,textActual)
```
Comment (nicolas-postigo) [inline staff/lucas-wloch/challenges/calculadora-app/main.js]: (,) are missing

File: staff/lucas-wloch/challenges/memo-test-app/index.html [hunk 10]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Memory-Test</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="main.js" defer></script>
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/memo-test-app/index.html]: script has to be under the body closure

File: staff/lucas-wloch/challenges/tic-tac-toe-app/index.html [hunk 11]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="main2.js" defer></script>
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/tic-tac-toe-app/index.html]: you have to change this, please put it under </body>

File: staff/lucas-wloch/challenges/tic-tac-toe-app/main2.js [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+        
+        jugando.textContent = this.jugando
+        
+
+    }
+    endgame() {
+        let result
+        if (buttons[0].textContent === buttons[1].textContent && buttons[1].textContent === buttons[2].textContent && buttons[2].textContent !== "") { result = `${"GANA "} ${buttons[0].textContent}`
+        jugando.textContent = result
+
```
Comment (PILARLATORREUSAN) [inline staff/lucas-wloch/challenges/tic-tac-toe-app/main2.js]: please remove, that empty line

File: staff/lucas-wloch/challenges/calculadora-app/main.js (hunk 9 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/main.js]: done thanks

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js (hunk 3 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/calculate.js]: done =)

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js (hunk 6 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/calculate.js]: done =)

File: staff/lucas-wloch/challenges/calculadora-app/main.js (hunk 8 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/main.js:9]: done =)

File: staff/lucas-wloch/challenges/tic-tac-toe-app/main2.js (hunk 12 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/tic-tac-toe-app/main2.js]: done =)

File: staff/lucas-wloch/challenges/calculadora-app/README.md (hunk 4 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/README.md]: Done Thanks =)

File: staff/lucas-wloch/challenges/calculadora-app/calculate.js (hunk 5 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/calculate.js]: Done

File: staff/lucas-wloch/challenges/memo-test-app/main.js (hunk 7 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/memo-test-app/main.js]: Done =)

File: staff/lucas-wloch/challenges/tic-tac-toe-app/index.html (hunk 11 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/tic-tac-toe-app/index.html]: Thanks

File: staff/lucas-wloch/challenges/memo-test-app/index.html (hunk 10 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/memo-test-app/index.html]: Done Thanks=)

File: staff/lucas-wloch/challenges/calculadora-app/index.html (hunk 2 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/index.html]: done

File: staff/lucas-wloch/challenges/calculadora-app/README.md (hunk 1 above)
Comment (lucaswloch) [reply staff/lucas-wloch/challenges/calculadora-app/README.md]: ok Done

**Review by simonbesteiro — APPROVED** [review]
review done

**Pull request merged**